api/src/test/java/bio/terra/tanagra/service/query/QueryServiceTest.java

```diff
@@ -1,7 +1,6 @@
 package bio.terra.tanagra.service.query;
 
-import static bio.terra.tanagra.service.underlay.NauticalUnderlayUtils.SAILOR;
-import static bio.terra.tanagra.service.underlay.NauticalUnderlayUtils.SAILOR_RATING;
+import static bio.terra.tanagra.service.underlay.NauticalUnderlayUtils.*;
```

Comment:
nit: static import `*` seems potentially problematic, general advice I've followed is to avoid all `*` imports in java

Reply:
SGTM. Some other test files doing that, I'll update incrementally or all at once later.